Runtime/CourseController/AdvancedCourseController.cs

```diff
@@ -458,7 +453,13 @@ private void SetupLanguagePicker()
             });
             
             // If there is only one option, the dropdown is currently disabled.
-            languagePicker.enabled = supportedLanguages.Count > 1;
+            if (supportedLanguages.Count <= 1)
```

Comment:
Anyway we can avoid the same 7 lines 3 times in one file?
And it is the same for the Standalone Controller

Reply:
Don't think that this will look better because the only line that is identical in that 7 lines blocks is:
colorBlock.normalColor = Color.gray;

Reply:
The rest has all different variables